scripts/containers/HomePage.js

```diff
@@ -0,0 +1,22 @@
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+
+import HomePage from "../components/HomePage";
+import * as PollActions from "../actions/poll";
+
+
+function mapStateToProps(state) {
+  return {
+    server: "https://kinto-leplatrem.herokuapp.com/v1",
+    bucket: "happiness"
```

Comment:
This should really go to a reducer, see previous comments made on the `Homepage` comp local state handling.